src/components/common/ProductBox/ProductBox.module.scss

```diff
@@ -24,13 +24,13 @@
 
     .images {
       position: absolute;
-      size: cover;
       padding: 0;
       top: 0;
       left: 0;
       height: 100%;
       width: 100%;
       z-index: 0;
+      size: cover;
```

Comment:
Ta linijka kodu nic nie robi, size: nie może mieć value "cover".
https://www.w3.org/TR/css-page-3/#page-size-prop

Reply:
> Ta linijka kodu nic nie robi, size: nie może mieć value "cover". https://www.w3.org/TR/css-page-3/#page-size-prop

To jest do zadanie WDP220602-1, mimo to zostało już poprawione, został również dodany commit

Reply:
skoro do zadania 1 to czemu jest tutaj :)